### DISCUSSION TITLE
Welcome to eoAPI Discussions!

### DISCUSSION THREAD

**Discussion by vincentsarago** [developmentseed/eoAPI#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hrodmn** [comment]
Greetings eoAPI community! I am a data engineer at NCX where I maintain the geospatial data infrastructure. My goals are to improve accessibility and visibility of NCX's raster and vector data for internal- and (possibly) external use.

I have been searching for examples of STAC API + TiTiler deployment configurations because my current deployment is not very robust. I found eoAPI yesterday and after playing with it for a few hours I think I am going to replace our existing services with the excellent CloudFormation stack generated by this package.
